package.json

```diff
@@ -61,5 +61,8 @@
     "trailingComma": "all",
     "singleQuote": true,
     "arrowParens": "avoid"
+  },
+  "dependencies": {
+    "react-intersection-observer": "^8.31.0"
   }
```

Comment:
I've added `react-intersection-observer` as a dependency because we probably don't want to maintain our own `IntersectionObserver` implementation. The implementation with `useInView()` was straight forwad.